src/main/sensors/esc_sensor.c

```diff
@@ -361,8 +361,10 @@ void escSensorProcess(timeUs_t currentTimeUs)
     }
 }
 
+#endif
+
 int calcEscRpm(int erpm)
 {
-    return (erpm * 100) / (motorConfig()->motorPoleCount / 2);
+    //return (erpm * 100) / (motorConfig()->motorPoleCount / 2);
```

Comment:
Please remove comment. As the revised calculation fits the new code, how about the existing code?
Is there a difference in calculation for ESC_SENSOR vs DSHOT_TELEMETRY?

Reply:
I checked the assembly code on m4 cortex with -O3 compiler optimization and for some reason the compiler doesn't simplify the division by 2. So the existing code is slower (it does an additional right hand, shift which is an integer division by 2).

Maybe we can change the comment to
```c
// (erpm * 100) / (motorConfig()->motorPoleCount / 2);
```
to have it on record where the eRPM to RPM conversion comes from?